modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/internal/servlet/taglib/BaseContainerTag.java

```diff
@@ -375,6 +375,26 @@ protected int processStartTag() throws Exception {
 		return EVAL_BODY_INCLUDE;
 	}
 
+	protected void renderIdAttribute(String _id) throws Exception {
```

Comment:
You need to use this method in [line 365](https://github.com/liferay-frontend/liferay-portal/pull/427/commits/135f0643d5da441dc8043bb58242999a70d77512#diff-2b4cdf2f6f59a9e1dcf930ab065f8d291606569ee9732953b52b52bff5b941eaR365), replacing that block there.

Reply:
@carloslancha Are these, id and class, methods supposed to have the parameters `String _id` and `String _cssClass` like I have them in this commit?

Reply:
You don't really need them since you can call `getId()` and `getCssClass` from there

Reply:
Anyway, if you rebase your branch with master this `_id` and `_cssClass` changes are already there ;)